specs/deposits.md

```diff
@@ -163,7 +173,7 @@ This transaction MUST have the following values:
    contract][predeploy]).
 3. `mint` is `0`
 4. `value` is `0`
-5. `gasLimit` is set to the maximum available.
+5. `gasLimit` is set to 75,000.
```

Comment:
Is this enforced by consensus or is it a policy? This should be made clear and if i understand correctly it is a policy

Reply:
It's a parameter in the rollup node. It causes a chain split if it's not set the same for every node (at least right now).

Reply:
Ah that makes sense, so its a consensus param